### ISSUE TITLE
Build failed due to ulp-go version missmatch

### ISSUE THREAD

**Issue by snip** [infinytum/ubnt-cloudkey-docker#2]
```
The following packages have unmet dependencies:
 unifi-core : Depends: ulp-go (>= 0.1.15-1061) but 0.1.11-1041 is to be installed
```
But it looks like the next version (1.2.13) is only available for arm64.

Any idea?

Thanks

**Comment by infinytum** [comment]
Yeah I totally have an idea... my sed command must be outdated... Lets see...

**Comment by infinytum** [comment]
Eh, not the only issue... Apparently Ubiquiti deleted the unifi-protect AMD64 package from their APT repository...

I guess their learned their lesson? XD

**Comment by andraghetti** [comment]
does that mean that there won't be any fix on this?

**Comment by infinytum** [comment]
I'll try. I need a new source for their AMD64 package though.

**Comment by infinytum** [comment]
I found a new source but the actual file is locked. Meh.

**Comment by andraghetti** [comment]
Locked how?

**Comment by infinytum** [comment]
I can retrieve a list of available AMD64 builds of unifi protect but the download link requires authentication, which I do not have.

**Comment by andraghetti** [comment]
Could it be just an ui.com account?
